decentralised-identity.md

```diff
@@ -0,0 +1,106 @@
+
+Decentralized identity is all about giving you control over what you share. Here's how it increases your privacy:
+
+- Selective Disclosure: Instead of handing out your entire identity like a business card, decentralized identity lets you be picky. You can choose specific details or credentials to share, keeping the rest under wraps. It's like showing your ID without revealing your home address.
+- Need-to-Know Basis: Only share what's necessary. If someone just needs to confirm your age, that's all they get. No unnecessary details get thrown into the mix, keeping your personal info on a need-to-know basis.
+- Reduced Data Collection: Decentralized identity cuts down on the data hoarding. Traditional systems often scoop up more info than they need. With decentralized identity, the focus is on minimalism, collecting only what's essential for a particular interaction.
+- User Empowerment: You're in charge. You decide who gets what, giving you the power to manage your digital identity. This approach puts a strong emphasis on user empowerment, ensuring that you're not at the mercy of entities collecting more data than necessary.
+- Mitigating Surveillance Risks: By minimizing the information shared, decentralized identity acts as a shield against unnecessary surveillance. It aligns with the cypherpunk principles of resisting unwarranted intrusion into personal lives.
+
+In a nutshell, decentralized identity lets you share just enough to get things done while keeping the rest safely tucked away. It's privacy on your terms.
+
+### Interoperability:
+
+Interoperability in decentralized identity is like making sure different digital systems speak the same language. It's about ensuring that your digital identity can smoothly work across various platforms and services without any hiccups.
+
+Imagine if your ID card only worked with one type of door lock. That would be inconvenient, right? Interoperability solves this issue. It means your decentralized identity can be recognized and used on different apps, websites, or services, just like your ID works wherever you go.
+
+Imagine buying things online without constantly entering your details. Using blockchain, your payment details become tamper-resistant and transparent. No single authority holds all your financial info, reducing the risk of a massive data breach. This isn't just about security; it's about convenience. With a decentralized identity, online payments become more streamlined, less prone to fraud, and entirely under your control. 
+
+The key is having common standards and rules that everyone follows. This way, whether you're logging into a social media account, accessing a financial service, or using an online marketplace, your decentralized identity seamlessly fits. It's like having a universal key for your digital interactions, making things more convenient and user-friendly. Interoperability is the glue that holds together the vision of a decentralized identity.
+
```

Comment:
Have added the following paragraph:

Would security be an issue?

Security remains a critical consideration, primarily as these rely heavily on technologies like blockchain and cryptographic methods. Potential vulnerabilities may arise from implementation flaws, coding errors, or attacks on the underlying infrastructure. The use of cryptographic keys, while fundamental for user control, introduces risks if not adequately managed, with potential concerns related to key loss or theft. Additionally, the distributed and decentralized nature of these systems may pose challenges in ensuring uniform security standards across the entire network.


I do think this article will get people talking. My point would be taking the positive view it can be achieved. The point about door locks is very good. Yes not sure I'd like these on-chain. Payment mechanisms should be achievable.